### ISSUE TITLE
Can't rename file

### ISSUE THREAD

**Issue by khedaywi** [meeech/Shopify-Theme-Tool-Public#5]
Renaming a file (i.e. template name, asset file name, etc.) results in the newly renamed file being uploaded as an entirely new file and the "old" named file remains.

Would be cool it it deletes the "old" file and uploads the "new" file with the new name...

**Comment by meeech** [comment]
yes. There is no file tracking.  
I wanted to avoid destructive actions like deleting from the shop as a preventative measure (accidentally deleting a live theme). Its pretty straightforward to add in that it deletes files that are deleted locally.

**Comment by khedaywi** [comment]
Yeah, it's not really too big of an issue, especially if you're working on a dev site. Would just be a nicety, especially for a live site; not having to reload the theme or go and hand delete the excess files, etc.
